rest-api/src/test/java/gov/cms/qpp/conversion/api/acceptance/CpcApiAcceptance.java

```diff
@@ -0,0 +1,154 @@
+package gov.cms.qpp.conversion.api.acceptance;
+
+import gov.cms.qpp.conversion.api.helper.JwtPayloadHelper;
+import gov.cms.qpp.conversion.api.helper.JwtTestHelper;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Map;
+
+import static com.google.common.truth.Truth.assertThat;
+import static io.restassured.RestAssured.get;
+import static io.restassured.RestAssured.given;
+import static io.restassured.RestAssured.put;
+
+@ExtendWith(RestExtension.class)
+class CpcApiAcceptance {
+
+	private static final String CPC_UNPROCESSED_FILES_API_PATH = "/cpc/unprocessed-files";
+	private static final String CPC_FILE_API_PATH = "/cpc/file/";
+
+	@BeforeAll
+	static void createUnprocessedItem() {
+		given()
+			.multiPart("file", Paths.get("../sample-files/CPCPlus_Success_PreProd.xml").toFile())
+			.when()
+			.post("/");
+	}
+
+	@Test
+	@Tag("acceptance")
+	void testNoSecurityForUnprocessedFiles() {
+		get(CPC_UNPROCESSED_FILES_API_PATH)
+			.then()
+			.statusCode(403);
+	}
+
+	@Test
+	@Tag("acceptance")
+	void testUnprocessedFiles() {
+
+		List<Map> responseBody = getUnprocessedFiles();
+
+		assertThat(responseBody).isNotEmpty();
+		assertThat(responseBody.get(0)).containsKey("fileId");
+		assertThat(responseBody.get(0)).containsKey("filename");
+		assertThat(responseBody.get(0)).containsKey("apm");
+		assertThat(responseBody.get(0)).containsKey("conversionDate");
+		assertThat(responseBody.get(0)).containsKey("validationSuccess");
+	}
+
+	@Test
+	@Tag("acceptance")
+	void testNoSecurityGetFile() {
+
+		String firstFileId = getFirstUnprocessedCpcFileId();
+
+		get(CPC_FILE_API_PATH + firstFileId)
+			.then()
+			.statusCode(403);
+	}
+
+	@Test
+	@Tag("acceptance")
+	void testGetFile() {
+
+		String firstFileId = getFirstUnprocessedCpcFileId();
+
+		given()
+			.auth().oauth2(createCpcJwtToken())
+			.get(CPC_FILE_API_PATH + firstFileId)
+			.then()
+			.statusCode(200)
+			.contentType("application/xml");
+	}
+
+	@Test
+	@Tag("acceptance")
+	void testNoSecurityMarkFileUnprocessed() {
```

Comment:
Can we get a quick rename here?  You're marking the file as processed, not unprocessed.

Reply:
🤦‍♂️ Oops.